include/SFML/Graphics/Rect.inl

```diff
@@ -127,6 +127,14 @@ constexpr Vector2<T> Rect<T>::getSize() const
 }
 
 
+////////////////////////////////////////////////////////////
+template <typename T>
+constexpr Vector2<T> Rect<T>::getCenter() const
+{
+    return getPosition() + getSize() / T{2};
```

Comment:
The idea of `T{2}` is that it works for all types without possible warnings (`int`, `unsigned int`, `float`, `double`, ...)?

Reply:
Yes 🙂

Reply:
That's right, that's to ensure we're dividing by the same type as the `Vector2` being returned by `getSize()`. It's required to ensure compilation succeeds.